ProcessLib/RichardsMechanics/RichardsMechanicsFEM-impl.h

```diff
@@ -1535,10 +1541,15 @@ void RichardsMechanicsLocalAssembler<ShapeFunctionDisplacement,
 
         GlobalDimMatrixType const K_over_mu = k_rel * K_intrinsic / mu;
 
+        auto const phi = _ip_data[ip].porosity;
```

Comment:
Same as above, why not
```suggestion
        auto const& phi = _ip_data[ip].porosity;
```